src/indexnotation/tensormacros.jl

```diff
@@ -11,17 +12,35 @@ end
 
 """
     @tensor(block)
+=======
+    @tensor(block [, order = (...)])
+>>>>>>> eae44d1... add optional contraction order and corresponding treebuilder
```

Comment:
unresolved conflict?

Reply:
Strange, I don't have these lines locally.

Reply:
Ah, apparently I forgot to save a file.